src/test/resources/TestCaseforProviderIdentityInconsistentProviders3Stars.ttl

```diff
@@ -0,0 +1,73 @@
+@prefix owl:   <http://www.w3.org/2002/07/owl#> .
+@prefix sq-a:  <http://w3id.org/squirrel/activity#> .
+@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sq-m:  <http://w3id.org/squirrel/metadata> .
+@prefix sq-g:  <http://w3id.org/squirrel/graph#> .
+@prefix sq-s:  <http://w3id.org/squirrel/status#> .
+@prefix dct:   <http://purl.org/dc/terms/> .
+@prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix dcat:  <http://www.w3.org/ns/dcat#> .
+@prefix sq-w:  <http://w3id.org/squirrel/worker#> .
+@prefix prov:  <http://www.w3.org/ns/prov#> .
+@prefix sq:    <http://w3id.org/squirrel/vocab#> .
+@prefix dct:    <http://purl.org/dc/terms/> .
+
+<http://projekt-opal.de/catalog/europeandataportal>
```

Comment:
Please remove the edited test files

Reply:
Reason: If we merge/integrate all metric implementaitons, we could become tons of files and nobody knows, how they appeared.

Reply:
Yes, agreed. I removed the unreal test files.